src/datalink/bpf.rs

```diff
@@ -19,17 +19,17 @@ use std::sync::Arc;
 use bindings::bpf;
 use packet::Packet;
 use packet::ethernet::{EthernetPacket, MutableEthernetPacket};
-use datalink::DataLinkChannelType;
+use datalink::{DataLinkChannelType,DataLinkSender,DataLinkReceiver,DataLinkChannelIterator};
```

Comment:
Spaces after the commas please.

Relatedly, I hope to integrate `rustfmt` into the build process in future so that style issues like this don't matter.